### PR TITLE
Set up basic site check funtionality for manual checks fixes #7

### LINKED ISSUE #7: As an administrator, I should be able to run a site check manually.

### PR DESCRIPTION
**Pull request by drapergeek**

### CODE REVIEW

**Comment by brentm5** [comment]
what do you mean about web named methods?

**Comment by drapergeek** [comment]
Well named methods*

**Comment by drapergeek** [comment]
Need to use proper commit messages:
https://github.com/thoughtbot/guides/tree/master/protocol#write-a-feature

Check out "write a good commit message".

I know it seems anal but lets follow best practices.

**Comment by brentm5** [comment]
I removed the instance variables.  It is good by my standards so you can merge if you give it the final checkover

File: app/controllers/admin/manual_site_checks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Admin::ManualSiteChecksController < ApplicationController
+  def show
+    MonitoredService.all.each do |service|
+      3.times do
+        ServiceChecker.new(service).execute
+      end
+    end
```
Comment (drapergeek) [inline app/controllers/admin/manual_site_checks_controller.rb:7]: white space after the block

File: spec/requests/services/test_services_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+# as an adminstrator
```
Comment (drapergeek) [inline spec/requests/services/test_services_spec.rb]: remove comments

File: app/controllers/admin/manual_site_checks_controller.rb (hunk 1 above)
Comment (drapergeek) [inline app/controllers/admin/manual_site_checks_controller.rb:7]: Fixed.

**Pull request merged**